definitions/private-law/json/ComplexTypes/ManageOrders/ComplexTypes.json

```diff
@@ -62,6 +62,15 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
+  {
+    "LiveFrom": "18/12/2023",
+    "ID": "OtherOrderDetails",
+    "ListElementCode": "orderCreatedByEmailId",
```

Comment:
Recently this changes was merged to master of storing the creator email id, please check if thats present so you can remove this.

Reply:
The change in master is for name not for the email